notify-api/src/notify_api/resources/sms.py

```diff
@@ -32,4 +32,5 @@ def post():
         """Send notification."""
         # sms_payload = request.get_json(force=True)
         # get_sms_service().send(sms_payload)
+        # pylint: disable=W0719
```

Comment:
For each of these unimplemented methods, could we add a TODO for us to update them and remove the generic `Exception` at some point?

Reply:
Added TODO as suggested. Thanks.